src/gsi/read_obs.F90

```diff
@@ -1477,7 +1477,16 @@ subroutine read_obs(ndata,mype)
                    call read_hdraob(nread,npuse,nouse,infile,obstype,lunout,twind,sis,&
                         prsl_full,hgtl_full,nobs_sub1(1,i),read_rec(i))
                    string='READ_UPRAIR'
+                else if (index(infile,'wbbufr') /=0)then
+                        call read_gsb(nread,npuse,nouse,infile,obstype,lunout,gstime,twind,sis,&
+                         nobs_sub1(1,i),read_rec(i))
+                    string='READ_GSB'
+                else if (index(infile,'sdbufr') /=0)then
+                        call read_saildrone(nread,npuse,nouse,infile,obstype,lunout,gstime,twind,sis,&
+                         nobs_sub1(1,i),read_rec(i))
+                    string='READ_SAILDRONE'
                 else
+                       write(*,*) 'Reading Prepbufr ',infile
```

Comment:
Is this a leftover debug `write`?

Reply:
@RussTreadon-NOAA Good catch! I will delete.